imports/util/email-invite.js

```diff
@@ -1,17 +1,38 @@
 import { Email } from 'meteor/email'
 import url from 'url'
+import bugzillaApi from '../util/bugzilla-api'
+const { callAPI } = bugzillaApi
 
-export const invite = (user, invitedBy) =>
-  Email.send({
-    to: user.emails[0].address,
-    from: process.env.FROM_EMAIL,
-    subject: `Invite by ${invitedBy.profile.name || invitedBy.emails[0].address}`,
-    text: `Hi there,
+export const invite = (user, invitedBy) => {
+  try {
+    const caseId = user.invitedToCases[0].caseId
+    const unitId = user.invitedToCases[0].unitId
+    const caseData = callAPI('get', `/rest/bug/${caseId}`, {}, true, true)
+    const unitData = callAPI('get', `/rest/product/${unitId}`, {}, true, true)
+    const caseTitle = caseData.data.bugs[0].summary
+    const unitName = unitData.data.products[0].name.trim()
+    const unitDesc = unitData.data.products[0].description.replace(/[\n\r]+/g, ' ').trim()
+    const invitorUsername = invitedBy.profile.name
+    const invitorEmailAddress = invitedBy.emails[0].address
+    const inviteeRole = user.invitedToCases[0].role
+    const invitorRole = unitData.data.products[0].components.find(
+      ({default_assigned_to: defAssigned}) => defAssigned === invitedBy.emails[0].address
+    )
+    const roleStr = invitorRole ? invitorRole.name : 'Administrator'
 
-You have been invited to contribute to a case.
+    Email.send({
+      to: user.emails[0].address,
+      from: process.env.FROM_EMAIL,
+      replyTo: `${invitorUsername} <${invitorEmailAddress}>`,
+      subject: `New Case: ${caseTitle}`,
+      text: `Hi there,
 
-Please click the link below to get started:
+${invitorUsername || invitorEmailAddress}, the ${roleStr} for the Unit ${unitName} (${unitDesc}) has invited you to collaborate on the case ${caseTitle} as the ${inviteeRole} for that unit.
+
+Please click the link below to get more information about the case and reply to ${invitorUsername || 'him'}:
 ${url.resolve(process.env.ROOT_URL, `/invitation?code=${user.invitedToCases[0].accessToken}`)}
 
 Kind regards,
```

Comment:
How would the 'signature' be added to the email? isn't it missing?

Reply:
What do you mean by signature? Stuff after "Kind regards," ??

Reply:
yeah, that stuff :)